apps/monitoring_dashboard/jobs/semaphore.exs

```diff
@@ -0,0 +1,57 @@
+use Kitto.Job.DSL
+
+defmodule Kitto.Jobs.Semaphore do
```

Comment:
Hi all, Looking through this, I would recommend moving the modules that you're defining in the job files into the `lib/` directory. Having the API client in the same location as the interface for Kitto is kind of like merging controllers and views into the same modules. Moving the API clients into the `lib/` directory also helps in the future if you want to break those clients out into separate Hex projects.